modal/cli/app.py

```diff
@@ -247,3 +248,44 @@ async def history(
 
     rows = sorted(rows, key=lambda x: int(str(x[0])[1:]), reverse=True)
     display_table(columns, rows, json)
+
+
+@app_cli.command("dashboard", no_args_is_help=True)
+@synchronizer.create_blocking
+async def dashboard(
+    app_identifier: str = APP_IDENTIFIER,
+    *,
+    env: Optional[str] = ENV_OPTION,
+):
+    """Open the dashboard for a deployed App in the web default browser.
+
+    **Examples:**
+
+    Open the dashboard based on an app ID:
+
+    ```bash
```

Comment:
BTW using "bash" here causes the CLI help to look bad:
<img width="831" alt="image" src="https://github.com/user-attachments/assets/5b230793-ff30-48ed-a09e-d56fc64f8937">

Not sure if there's a better way to fix that but since there's not actually any bash syntax to mark up here it will be marginally cleaner to omit it.

Reply:
Yeah probably should — I only noticed what it does to the terminal help recently.